src/main/java/frc/robot/subsystems/Grabber.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import frc.robot.RobotMap;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Grabber extends SubsystemBase {
+    private CANSparkMax FRONT_LEFT_DRIVE_MOTOR = new CANSparkMax(RobotMap.FRONT_LEFT_DRIVE_MOTOR_CAN_ID, MotorType.kBrushless);
+    private CANSparkMax FRONT_RIGHT_DRIVE_MOTOR = new CANSparkMax(RobotMap.FRONT_RIGHT_DRIVE_MOTOR_CAN_ID, MotorType.kBrushless);
+    private CANSparkMax BACK_LEFT_DRIVE_MOTOR = new CANSparkMax(RobotMap.BACK_LEFT_DRIVE_MOTOR_CAN_ID, MotorType.kBrushless);
+    private CANSparkMax BACK_RIGHT_DRIVE_MOTOR = new CANSparkMax(RobotMap.BACK_RIGHT_DRIVE_MOTOR_CAN_ID, MotorType.kBrushless);
+
+    public void intake() {};
+    public void scoreHigh() {};
+    public void scoreMid() {};
+    public void scoreLow() {};
```

Comment:
I can think of an action that we'll definitely want to be able to do that isn't here; see if you can figure it out. Refer back to old robots if necessary.

Reply:
After looking at the old robot code, I'm not entirely sure, since the intake is a lot more complicated than our grabber, but could we be needing an eject function?

Reply:
Yeah. Whatever you want to call it; eject, outtake, or something else. Even if it ends up being the same code as scoring low (as it was on piggie smalls) it should still be its own function here.